scipy/optimize/tests/test_zeros.py

```diff
@@ -58,5 +59,15 @@ def test_deriv_zero_warning(self):
         assert_warns(RuntimeWarning, cc.newton, func, 0.0, dfunc)
 
 
+class TestRootResults:
+    def test_repr(self):
+        _, r = zeros.bisect(functions[0], .5, sqrt(3),
+                            xtol=0.1e-12, full_output=True)
+        expected_repr = ("      converged: True\n           flag: 'converged'"
+                         "\n function_calls: 46\n     iterations: 44\n"
+                         "           root: 0.999999999999985")
+        assert_string_equal(r.__repr__(), expected_repr)
```

Comment:
Since this is a Python special method, `repr(a)` could be preferred over `a.__repr__()`